Mathlib/Probability/Moments.lean

```diff
@@ -331,6 +357,39 @@ theorem measure_le_le_exp_cgf [IsFiniteMeasure μ] (ε : ℝ) (ht : t ≤ 0)
   rw [exp_add]
   exact mul_le_mul le_rfl (le_exp_log _) mgf_nonneg (exp_pos _).le
 
+open Measurable
+
+theorem mgf_gaussian
+(X: Ω → ℝ) (hXm: Measurable X) (hX: (Measure.map X μ) = (gaussianReal 0 1)) (t : ℝ) :
```

Comment:
Can you also fix the indentation of this one? Also, I'd move it to the files on gaussians, to avoid putting more imports in the current file (which should remain one of the basic files in probability). Maybe also give the statement for non-centered non-normalized gaussians?